packages/components/src/composite/unstable/stories/index.story.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { isRTL } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Composite,
+	CompositeGroup,
+	CompositeItem,
+	useCompositeState,
+} from '..';
+import Legacy from '../../legacy/stories/index.story';
+
+type InitialState = Parameters< typeof useCompositeState >[ 0 ];
+
+const Placeholder = ( _: InitialState ) => <></>;
+
+const meta: Meta< typeof Placeholder > = {
+	title: 'Components/Composite/Composite (Unstable)',
+	id: 'components-composite-unstable',
+	component: Placeholder,
+	args: {
+		rtl: isRTL(),
+		loop: false,
+		shift: false,
+		wrap: false,
+	},
+	argTypes: { ...Legacy.argTypes },
+};
+export default meta;
+
+export const TwoDimensionsWithStateProp: StoryFn< typeof Placeholder > = (
+	initialState
+) => {
+	const state = useCompositeState( initialState );
+	// eslint-disable-next-line no-console
+	console.log( { initialState, state } );
```

Comment:
Is this `console.log` on purpose, or should it be cleaned up?

Reply:
Hrm. Will clean up.